upf/upf_cli.c

```diff
@@ -455,6 +455,8 @@ upf_tdf_ul_table_add_del_command_fn (vlib_main_t * vm,
   if (table_id == ~0)
     return clib_error_return (0, "table-id must be specified");
 
+  upf_proxy_init (vm);
```

Comment:
why does this cli command the proxy to be ready? I don't see that is has a relation to the proxy process.
And if it does, then the call need to go into vnet_upf_tdf_ul_table_add_del.

Reply:
Fixed proxy init, now this is no longer needed and thus removed